system/doc/oam/oam.md

```diff
@@ -50,7 +50,34 @@ client-server model:
 
 The following picture illustrates the idea:
 
-![Terminology](assets/terminology.gif "Terminology")
+```mermaid
```

Comment:
Please restore everything in the OAM guide to the way it was originally. We can have another go later, but for now leave it as is.